docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -14,117 +16,157 @@ This how-to guide is still a work in progress and will change.
 It's currently very specific to VSHN and needs further changes to be more generic.
 --
 
+
 == Prerequisites
-* cloudscale.ch API token
-* `terraform`
-* `mc` https://docs.min.io/docs/minio-client-quickstart-guide.html[Minio client]
+
+* `docker`
+* `docker-compose`
+* `mc` https://docs.min.io/docs/minio-client-quickstart-guide.html[Minio client] (aliased to `mc` if necessary)
 * `jq`
 * `yq` https://mikefarah.gitbook.io/yq[yq YAML processor]
 * `openshift-install` https://cloud.redhat.com/openshift/install/metal/user-provisioned[OpenShift Installer]
-* Working local https://github.com/projectsyn/commodore[Commodore] setup
+* https://github.com/projectsyn/commodore[Commodore] docker-compose setup
 
 
 == Cluster Installation
 
-. Register the new OpenShift 4 cluster in Lieutenant: https://control.vshn.net/syn/lieutenantclusters
+Register the new OpenShift 4 cluster in Lieutenant: https://control.vshn.net/syn/lieutenantclusters
+
+=== Configure input
 
-. Configure input
-+
 [source,console]
 ----
 export CLOUDSCALE_TOKEN=<cloudscale-api-token> # From https://control.cloudscale.ch/user/api-tokens
 export GITLAB_TOKEN=<gitlab-api-token> # From https://git.vshn.net/profile/personal_access_tokens
-export GITLAB_CATALOG_PROJECT_ID=<project-id> # GitLab project ID of the catalog repo
-export REGION=rma # rma or lpg
-export CLUSTER_ID=<cluster-id>
-export TENANT_ID=<tenant-id>
-export BASE_DOMAIN=ocp4-poc.appuio-beta.ch
-export PULL_SECRET=<redhat-pull-secret> # From https://cloud.redhat.com/openshift/install/pull-secret
+export GITLAB_CATALOG_PROJECT_ID=<project-id> # GitLab numerical project ID of the catalog repo
+export REGION=rma # rma or lpg (without the zone number)
+export CLUSTER_ID=<lieutenant-cluster-id>
+export TENANT_ID=<lieutenant-tenant-id>
+export BASE_DOMAIN=ocp4-poc.appuio-beta.ch # See xref:explanation/dns_scheme.adoc[DNS Scheme]
```

Comment:
```suggestion
export BASE_DOMAIN=appuio-beta.ch # See xref:explanation/dns_scheme.adoc[DNS Scheme]
```

The subdomain `ocp4-poc` shall only be used for clusters placed within the corresponding GCP project.